docs/DeveloperGuide.md

```diff
@@ -899,23 +892,24 @@ feature freeze. The plans to improve our features are as such.
 
 1. There is a small GUI bug that causes the overflow of names for Tags and Weddings which causes cosmetic flaws.
 Planned enhancements would be to truncate the name with `...`.
-2. There is a duplicate validation bug that allows the creation of certain "duplicate" Person, Wedding, Tag, Task, Tags.
+2. There is a duplicate validation bug that allows the creation of certain "duplicate" Person, Wedding, Task and Tags.
```

Comment:
Just wondering, is there a reason for everything to be singular except for Tags?

Reply:
I think it was just a typo, thanks for pointing out